cql3/single_column_relation.cc

```diff
@@ -58,6 +58,26 @@ single_column_relation::new_EQ_restriction(data_dictionary::database db, schema_
     return r;
 }
 
+expression
+static prepare_in_bind_marker(const std::vector<lw_shared_ptr<column_specification>>& receivers,
+                              const expr::expression& raw,
+                              data_dictionary::database db,
+                              const sstring& keyspace,
+                              prepare_context& ctx) {
+    assert(receivers.size() == 1);
```

Comment:
on_internal_error() is safer, it does not crash the server on a bug.

Reply:
Changed